suite-native/device/src/hooks/useDeviceConnect.tsx

```diff
@@ -74,12 +73,21 @@ export const useDeviceConnect = () => {
 
     const { translate } = useTranslate();
 
+    // const [isFirmwareSupported, setIsFirmwareSupported] = useState<boolean>(true);
+
+    const firmwareVersion = getFirmwareVersionArray(currentDevice);
+
     const handleDisconnect = useCallback(() => {
         if (currentDevice) {
             dispatch(deviceActions.deviceDisconnect(currentDevice));
         }
     }, [currentDevice, dispatch]);
 
+    const isFirmwareSupported = useMemo(
```

Comment:
Why are you memoizing this value? Is this operation computationally heavy?

I will recomputed on every rerender anyway since it is depending on a un-memoized `firmwareVersion`.

Reply:
b6eef5a8b